subprojects/logging/src/main/java/org/gradle/internal/logging/config/LoggingRouter.java

```diff
@@ -21,4 +21,5 @@
 
 public interface LoggingRouter extends LoggingSystem, LoggingOutputInternal {
     void configure(LogLevel logLevel);
+    void configureMaxWorkerCount(int maxWorkerCount);
```

Comment:
Do you think we should rename `configure(LogLevel)` to `configureLogLevel(LogLevel)`? Both configure method would be more aligned together.